federation/commons/src/main/java/org/codice/ddf/admin/commons/sources/SourceHandlerCommons.java

```diff
@@ -38,6 +38,8 @@
 
 public class SourceHandlerCommons {
 
+    private static final RequestUtils REQUEST_UTILS = new RequestUtils();
```

Comment:
There's nothing wrong with this - `RequestUtils` is threadsafe - but it seems weird to have a statically shared instance like that. A final per instance would be a little less peculiar.